ios/brave-ios/Sources/BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -574,6 +581,25 @@ public class SendTokenStore: ObservableObject, WalletObserverStore {
     addressError = sendAddress.isBTCAddress(isMainnet: isMainnet) ? nil : .notBtcAddress
   }
 
+  private func validateZcashSendAddress(
+    fromAccount: BraveWallet.AccountInfo,
+    recipient: String
+  ) {
+    Task { @MainActor in
+      let (_, zcashAddressError) = await zcashWalletService.transactionType(
+        accountId: fromAccount.accountId,
+        testnet: false,
```

Comment:
Shouldn't this be if the account is a testnet account? Rather than hardcoded false

Reply:
fixed in [c42351f](https://github.com/brave/brave-core/pull/28897/commits/c42351f8105f0d0c205c6555e9302a6d9f3c8573)